app.js

```diff
@@ -48,7 +52,7 @@ app.use(
 app.use(session({
   resave: false,
   saveUninitialized: false,
-  secret: process.env.SESSION_SECRET
```

Comment:
`SESSION_SECRET` should probably be in the .env[.sample]; we don't want to hard code the session secret.